app/src/components/governance/PageProposal.vue

```diff
@@ -10,14 +10,18 @@
       />
 
       <ProposalStatusBar
-        v-if="tallyHasValues"
+        v-if="tallyHasValues && !isPolkadotDemocracy"
```

Comment:
let's unhide it and just make sure the bar shows 100% if it's 100% or over

Reply:
why? imagine kickstarter. 4000% funded

Reply:
I obeyed xD

Reply:
how do you show 400% on a bar? the bar can only be 100% full - so show it as 100% full and say 400% !!

Reply:
was it problem before that the bar was 400% width?

Reply:
no i think it was fine before - now we'll unhide it and all is well. nothing to worry about here!